src/Content/Runtime/Settings.ts

```diff
@@ -101,6 +101,13 @@ class Settings {
 					type: 'checkbox',
 					defaultValue: true
 				},
+				{
+					id: 'general.emote_click',
+					label: 'Click emote to use',
+					hint: 'Click on any emote in chat to automatically input it in your chat draft',
+					type: 'checkbox',
+					defaultValue: true
```

Comment:
Unsure if this should be enabled by default as it removes the default functionality of opening the emote card.

Reply:
True, will need to find a way to prevent emote card behaviour when this option is on.